dtp/src/utils.rs

```diff
@@ -404,15 +339,153 @@ pub(crate) async fn gen_wallets_transfer_tssc(
         wallet_addresses.push(pub_key);
         wallet_priv_keys.push(priv_key);
 
-        // transfer funds
-        // TODO: send as bundle outside the for-loop. create a array of signed tx in this loop.
-        transfer_tssc(client.clone(), &funder_wallet, pub_key, U256::from(funding_amount))
-            .await
-            .expect(&format!("error in sending fund to {}", pub_key));
+        // M-1: transfer funds using API call
+        // Recommended for sending each transfer as tx. Hence, funder needs to sign for each transfer.
+        // transfer_tssc_single(client.clone(), &funder_wallet, pub_key, U256::from(funding_amount))
+        //     .await
+        //     .expect(&format!("error in sending fund to {}", pub_key));
     }
+
+    println!(
+        "\nCalling \'Fund\' contract\'s \'transfer_tssc_to_many\' \nmethod for sending funds in bulk..."
+    );
+
+    // M-2: transfer funds using 'Fund' contract
+    // Recommended for bulk transfer from single account.
+    // Also all transfers added via single tx i.e. funder's single signature.
+    transfer_tssc_bulk(
+        client.clone(),
+        &funder_wallet,
+        wallet_addresses.clone(),
+        U256::from(funding_amount),
+        fund_contract_addr,
+    )
+    .await?;
+
     Ok(signers)
 }
 
+/// Transfer TSSC to single account via eth API call method
+/// NOTE: For multiple receiver accounts, we can also use
+/// the contract "Fund"'s `transferTsscToMany`
+/// One can also leverage the contract "Fund"'s `transferTsscToSingle`
+/// function.
+pub(crate) async fn transfer_tssc_single(
```

Comment:
this can probably be removed.

Reply:
I mean remove the function completely, I don't think we will use it.

Reply:
Done